Gemfile

```diff
@@ -37,22 +37,28 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
-gem 'listen', '>= 3.0.5', '< 3.2'
+# Needs platform to avoid x86_64 in Gemfile.lock
+#gem 'nokogiri', platforms: :ruby
+gem 'nokogiri', platforms: [:mri, :mingw, :x64_mingw]
```

Comment:
`platforms: :ruby` seems to be correct.

Reply:
In fact, does nokogiri have to be specified at all?  Or does that depend on the version of ruby and/or bundler?

Reply:
Like I sait, this is WIP. I'm trying various combinations to get to the desired result.

I'like to avoid manual changes to `Gemfile.lock` as much as possible.

Reply:
> `platforms: :ruby` seems to be correct.

I'm afraid neither of those results in non-x86_64 lock file compatibility.

Reply:
Version specification seems to do the trick. Let me check if I can remove `mini_portile2` again.

Reply:
It's not possible: https://git.io/J3ok4

Additionaly, I'll look into some lock file "platform" handling.

Reply:
We're not seeing the same thing; what version of ruby and builder are you using here?

Reply:
I'm using Ruby 3.0 from yet unreleased rh-ruby30. I've also managed to work around the issues via some lock file handling (scripts attached).